packages/core/package.json

```diff
@@ -80,6 +80,7 @@
     "@envelop/testing": "^6.0.0",
     "@faststore/eslint-config": "^2.2.0-alpha.1",
     "@faststore/lighthouse": "^1.12.32",
+    "@faststore/cli": "^2.2.0-alpha.1",
```

Comment:
should we update it after the merge of this PR, right?

Reply:
I *think* lerna handles it automatically